lib/version.ex

```diff
@@ -31,7 +31,7 @@ defmodule PaperTrail.Version do
       )
     end
 
-    timestamps(updated_at: false)
+    timestamps(updated_at: false, type: Application.get_env(:paper_trail, :timestamps_type, :naive_datetime))
```

Comment:
Hi there, thanks for this contribution! Could you also add the respective datetimes into the migration generated on:
`lib/mix/tasks/papertrail/install.ex#28`?

Reply:
https://github.com/izelnakri/paper_trail/pull/108/commits/e3d89dd7fdfb553d3821e7e71ef381737d123e85

As you requested I've started to use the configuration in the migration.

Reply:
:+1: Could you also make the default `:timestamps_type` same for both? Then we can merge this once tests pass and cut a release ;)

Reply:
Sure, ECTOs [default option](https://hexdocs.pm/ecto/Ecto.Schema.html#timestamps/1) for this setting is `:naive_datetime` changing the default value for in the generated migration for backward compatibility.

Reply:
Changed in https://github.com/izelnakri/paper_trail/pull/108/commits/69b35366fb202c9735ecc82700cba2de58cb7036

Reply:
yes lets have it `:utc_datetime` as the default value, then I'll cut a minor release, and thanks for this feature!

Reply:
Ok, added in https://github.com/izelnakri/paper_trail/pull/108/commits/f07bf624550104be37678d40599c81e18bd94ded